src/coverage.ts

```diff
@@ -12,14 +12,24 @@ interface Props {
   summary: LcovSummary
   context: ContextInfo
   diffCoverage: DiffCoverageOutput
+  shouldAddAnnotation: boolean
+  threshold: number
+  differenceThreshold: number
+  patchThreshold: number
+  emptyPatch: boolean
 }
 
 export const coverage = async ({
   token,
   ghToken,
   summary,
   context,
-  diffCoverage
+  diffCoverage,
+  shouldAddAnnotation = true,
```

Comment:
```suggestion
  shouldAddAnnotation = false,
```

to match the intended default value

Reply:
i changed the action.yml instead.